api/envoy/extensions/filters/http/mcp/v3/mcp.proto

```diff
@@ -14,11 +15,27 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // MCP filter :ref:`configuration overview <config_http_filters_mcp>`.
 // [#extension: envoy.filters.http.mcp]
 
+// This filter will inspect and get attributes from MCP traffic.
 message Mcp {
-  // TODO: Add configuration fields
+  // Traffic handling mode for non-MCP traffic.
+  enum TrafficMode {
+    // Proxies the HTTP request and response without MCP spec check.
+    // This is the default mode.
+    PASS_THROUGH = 0;
```

Comment:
I think that pass-through is ambiguous term here, and I suggest the following:
mark this filter as work-in-progress (`option (xds.annotations.v3.file_status).work_in_progress = true;` at the file level), and we'll change these things later.